project/src/main/java/com/google/sps/utilities/SpannerTasks.java

```diff
@@ -701,4 +701,23 @@ public Void run(TransactionContext transaction) throws Exception {
     }
     return results;
   }
+
+  /**
+   * Delete search index entries for given event ID.
+   *
+   * @param eventId eventId to delete search entries for
+   */
+   public static void deleteIndexEntriesByEventId(String eventId) {
+    SpannerClient.getDatabaseClient()
+        .readWriteTransaction()
+        .run(
+            new TransactionCallable<Void>() {
+              @Override
+              public Void run(TransactionContext transaction) throws Exception {
+                String sql = String.format("DELETE FROM Results WHERE EventID=\"%s\";", eventId);
```

Comment:
I think in-lining this SQL call would be pretty reasonable here, since it's clear that the String is a SQL call.

Reply:
Done - inlined.